bin/watch-and-test-levelbuilder

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+require_relative '../deployment.rb'
+require_relative '../tools/hooks/hooks_utils.rb'
+require 'time'
+
+REPO_DIR = File.expand_path('../../', __FILE__)
+LEVELS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/scripts', __FILE__)
+LEVELBUILDER_CI_LOCK_FILENAME = 'levelbuilder_ci_lock'
+LEVELBUILDER_CI_LAST_RUN_FILENAME = 'levelbuilder_ci_last_run'
+
+Dir.chdir(REPO_DIR)
+
+exit if File.exist? LEVELBUILDER_CI_LOCK_FILENAME
+
+system "touch #{LEVELBUILDER_CI_LOCK_FILENAME}"
+
+last_modified_time = File.exist?(LEVELBUILDER_CI_LAST_RUN_FILENAME) ? DateTime.parse(File.read(LEVELBUILDER_CI_LAST_RUN_FILENAME)).to_time : Time.at(0)
+
+modified_files = HooksUtils.get_modified_files
+unstaged_files = HooksUtils.get_unstaged_files
+
+all_interesting_files = modified_files | unstaged_files
+
+level_file_modified = false;
+
+all_interesting_files.each do |filename|
+  if (filename.start_with? LEVELS_DIR) && File.mtime(filename) > last_modified_time
+    level_file_modified = true
+  end
+end
+
+unless level_file_modified
+  puts 'No new changes, cleaning up'
+  File.delete LEVELBUILDER_CI_LOCK_FILENAME
+  exit 0;
+end
+
+puts "Modifications to levelbuilder files detected - running testing"
+
+Dir.chdir(REPO_DIR + '/dashboard')
+
+test_result = system 'rake test'
```

Comment:
I think it's a good sanity check, but in next checking I was going to reduce the set of tests we run and add a set of UI tests for something more comprehensive